index.html

```diff
@@ -65,227 +80,223 @@ <h2 class="header-title ">I'm Abigail T Glad to see you!
+      </div>
+      <div class="card-content">
+        <h2 class="card-title">Tonic</h2>
+        <div class="card-info">
+          <p class="card-info-item-canopy">CANOPY</p>
+          <img class="circle-img" src="./images/circle.png" alt="Circle">
+          <p class="card-info-item">Back End Dev</p>
+          <img class="circle-img" src="./images/circle.png" alt="Circle">
+          <p class="card-info-item">2015</p>
+        </div>
+
+        <p class="card-description">
+          A daily selection of privately personalized reads; no accounts or sign-ups required..
+        </p>
+
+        <ul class="card-list">
+          <li class="card-list-tags-html">html</li>
+          <li class="card-list-tags-css">css</li>
+          <li class="card-list-tags-js">Javascript</li>
+        </ul>
+
+        <a class="card-link" href="#">See Project</a>
+
+      </div>
     </div>
-  </article>
+</article>
 
+
+ 
  <!-- My Second design -->
 
-  <article class="second-design">
-      <div class="design-outer">
-        <ul class="design-inner">
-          <li>
-            <div class="first-card">
-              <img
-                class="first-work-one"
-                src="images/second-work.png"
-                alt="First Work"
-              />
-              <div class="card-content">
-                <h3 class="card-title">Multi-Post Stories</h3>
-                <div class="card-info">
-                  <p class="card-info-item-canopy">CANOPY</p>
-                  <img class="circle-img" src="./images/circle.png" alt="Circle">
-                  <p class="card-info-item">Back End Dev</p>
-                  <img class="circle-img" src="./images/circle.png" alt="Circle">
-                  <p class="card-info-item">2015</p>
-                </div>
-
-                <p class="card-description">
-                  A daily selection of privately personalized reads; no accounts or sign-ups required.
-                </p>
-
-                <ul class="card-list">
-                  <li class="card-list-tags-html">html</li>
-                  <li class="card-list-tags-css">css</li>
-                  <li class="card-list-tags-js">Javascript</li>
-                </ul>
-
-                <a class="card-link" href="#">See Project</a>
-                
-              </div>
-            </div>
-          </li>
-        </ul>
+ <article class="my-design">
+  <div class="first-card">
+    <div class="card-image">
+      <img
+          class="first-work-one"
+          src="images/two.png"
+          alt="First Work"
+        />
+    </div>
+    <div class="card-content">
+      <h2 class="card-title">Multi-Post Stories</h2>
+      <div class="card-info">
+        <p class="card-info-item-canopy">FACEBOOK</p>
+        <img class="circle-img" src="./images/circle.png" alt="Circle">
+        <p class="card-info-item">Full Stack Dev</p>
+        <img class="circle-img" src="./images/circle.png" alt="Circle">
+        <p class="card-info-item">2015</p>
```

Comment:
:red_circle: Please make sure you increase the space between the project's picture and the project's description to match the Figma Design
As you can see below :arrow_down: Your design lacks enough space between the project picture and  description on the project cards where the project description is aligned to the right. You can use `nth-child(even)` to target these elements whose index is even  (the index of the first child is 1).



|Your Design|Figma  Design|
| :---: | :---:  |
|![y](https://user-images.githubusercontent.com/26736582/200081753-eb812cc6-8c62-4ee8-8f6e-7311b544fee0.png)|![f](https://user-images.githubusercontent.com/26736582/200081846-aef225c6-9ccb-458a-b005-e15a4ed0c5ae.png)|

Reply:
Conversation resolved. Thank you